datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -47,8 +47,9 @@
 )
 @click.option('--org-name', '-o', help='The org to use for data submission.')
 @click.option('--profile-memory', '-pm', is_flag=True, help='Whether to collect metrics about memory usage')
+@click.option('--dogstatsd', '-d', is_flag=True, help='Enable dogstatsd port on agent')
```

Comment:
(Nit) We use `-d` for "debug" elsewhere, could be confusing here… I'm not sure we actually need a shortcut?

```suggestion
@click.option('--dogstatsd', is_flag=True, help='Enable dogstatsd port on agent')
```

Reply:
Agreed - I forgot to take that shortcut out.